### PR TITLE
OD-838 [Fix] Fixed initialization for widget

### PR DESCRIPTION
**Pull request by skrupnyk**
@romanyosyfiv @inna-bieshulia

OD-838 https://weboo.atlassian.net/browse/OD-838

## Description
Added wait promise on init

## Backward compatibility
This change is fully backward compatible.

## Reviewers
@upplabs-alex-levchenko

### CODE REVIEW

File: js/build.js [hunk 1]
```diff
@@ -20,9 +20,12 @@ Fliplet.Widget.instance('list', function(data) {
     }
   });
 
-  var swipeToSaveLabel = data.swipeToSaveLabel || T('widgets.list.noImages.defaultListName');
+  Fliplet().then(function() {
+    $container.translate();
+    $container.translate({ swipeToSaveLabel: swipeToSaveLabel });
```
Comment (squallstar) [inline js/build.js:27]: It looks like you're using `swipeToSaveLabel` before it's declared. Why is that so and why do you need two specific calls of `translate` one after the other?

Comment (skrupnyk) [reply js/build.js:27]: > It looks like you're using `swipeToSaveLabel` before it's declared. Why is that so and why do you need two specific calls of `translate` one after the other?

Yes, I already noticed this problem. PRs will be updated soon.

File: js/build.js [hunk 2]
```diff
@@ -22,12 +22,14 @@ Fliplet.Widget.instance('list', function(data) {
...
-    window.ui['swipeSavedList' + $container.attr('data-list-uuid')] = new SwipeSaveList(this, {
-      savedListLabel: swipeToSaveLabel
-    });
-  }
+  Fliplet().then(function() {
+    $container.translate({ swipeToSaveLabel: swipeToSaveLabel });
+
+    if (data.swipeToSave) {
+      window.ui = window.ui || {};
+      window.ui['swipeSavedList' + $container.attr('data-list-uuid')] = new SwipeSaveList(this, {
```
Comment (tonytlwu) [inline js/build.js]: @skrupnyk The `this` variable in this line is likely referencing the wrong element, now that you've changed the context with `Fliplet().then(function() {})`

**Review by tonytlwu — APPROVED** [review]
@romanyosyfiv @inna-bieshulia This is ready for testing

Risk: High
Affected: Component user in apps
Verify: Component is correctly initialised and translated

Squash and merge: Use PR title

**Pull request merged**